run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+for i in `ls $(readlink -m $0/../tests)`
+do
+	echo -n Running test $i...
```

Comment:
Prefer `printf 'Running test %d' "$i"

Reply:
I somehow didn't know the shell had a printf command. Will do